_posts/2015-08-31-intro-to-daisywheeljs.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Adding the DaisywheelJS onscreen-keyboard to HTML5 Gamepad projects
+tags: daisywheeljs daisywheel gamepad html5
+username: likethemammal
+---
+
+### Some background
+
+HTML5 gaming has developed quite a bit since it first landed on the scene a few years back. Now, with proclamations from major browser vendors starting to pull support from Flash, and even [block some Flash ads(http://www.themonitordaily.com/google-chrome-adobe-autoplay/25687/), usage of HTML5’s latest technologies will spread further. Old Flash game devs will become new HTML5 game devs and entire games will be ported over for maintainability’s sake.
```

Comment:
Forgot a closing **]**. Should be `[block some Flash ads](http://www.themonitordaily.com/google-chrome-adobe-autoplay/25687/)`